config/manager/manager.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       containers:
       - args:
         - --leader-elect
-        image: ko://github.com/k8s-service-bindings/implementation
+        image: ko://github.com/k8s-service-bindings/service-binding-controller
```

Comment:
This is an unrelated change. Let me know if I should create a separate PR.

Reply:
PR: https://github.com/k8s-service-bindings/service-binding-controller/pull/26

Reply:
By 👍 I meant that the unrelated change was ok, but I merged #26 since it was ready to go :)